.github/actions/test-dotnet/Merge-FailureDumps.ps1

```diff
@@ -8,4 +8,4 @@ $rootDirectory = (Test-Path -Path $testDirectory) ? $testDirectory : $Directory
 Get-ChildItem $rootDirectory -Recurse |
     Where-Object { $PSItem.Name -eq 'FailureDumps' } |
     ForEach-Object { $PSItem.GetDirectories() } |
-    ForEach-Object { Move-Item $PSItem.FullName "$Directory/FailureDumps/${_.Name}" }
+    ForEach-Object { Move-Item $PSItem.FullName "$Directory/FailureDumps/$($PSItem.Name)" }
```

Comment:
FYI this is a functional change, not just cosmetic. Variable names enclosed in braces are used to be able to use special characters in the variable name, so it won't behave the same as `$_.Name` (or `$PSItem.Name`). I also removed usages of braced variables in other files.

Idea: Add an analyzer that detects that braced variables are used unnecessarily, which is basically when you don't have a special character enclosed in the braces.